lms/lmsdb/models.py

```diff
@@ -129,6 +129,8 @@ class Exercise(BaseModel):
     date = DateTimeField()
     users = ManyToManyField(User, backref='exercises')
     is_archived = BooleanField()
+    notebook_num = IntegerField(default=0)
```

Comment:
can you add a 'semi' migration like we did not in CommentText? look at bootstrap.py